src/Storage/Storage.Management/StorageAccount/NewAzureStorageAccount.cs

```diff
@@ -74,6 +74,7 @@ public class NewAzureStorageAccountCommand : StorageAccountBaseCmdlet
         [ValidateSet(AccountKind.Storage,
             AccountKind.StorageV2,
             AccountKind.BlobStorage,
+            AccountKind.BlockBlobStorage,
```

Comment:
@blueww would you mind regenerating the help for the cmdlet so this value is listed in the `AcceptedValues` section?

Reply:
I have regenerated the help with the change.